src/test/resources/application-test.yml

```diff
@@ -17,8 +17,6 @@ spring:
     username: cmd
     password: dummy
 
-  cache:
-    type: NONE
```

Comment:
removing this should be fine as long as integration tests use different params on the affected endpoint